src/components/sections/campaign/AllActionsPane.jsx

```diff
@@ -62,21 +64,60 @@ export default class AllActionsPane extends CampaignSectionPaneBase {
         else if (actionList.items.length == 0) {
             endDate = startDate.clone().addDays(8 * 7);
         }
-
+        
         if (this.state.viewMode == 'cal') {
             let actions = actionList.items.map(i => i.data);
-
+            
             viewComponent = <ActionCalendar actions={ actions }
-                    startDate={ startDate } endDate={ endDate }
-                    onSelectDay={ this.onSelectDay.bind(this) }
-                    onAddAction={ this.onCalendarAddAction.bind(this) }
-                    onMoveAction={ this.onCalendarMoveAction.bind(this) }
-                    onCopyAction={ this.onCalendarCopyAction.bind(this) }
-                    onSelectAction={ this.onSelectAction.bind(this) }/>
+            startDate={ startDate } endDate={ endDate }
```

Comment:
Try to avoid changes to whitespace unless there's actually something substantial to change.